react/features/base/react/components/NavigateSectionList.js

```diff
@@ -93,7 +93,7 @@ class NavigateSectionList extends Component<Props> {
 
         return (
             <SectionList
-                ListEmptyComponent = { renderListEmptyComponent() }
```

Comment:
This was fixing an issue and was intentional. With the change below the case of disconnected calendar doesn't work anymore, so I reverted this change.

Reply:
I have a fix I'm working on for it now, which should try to fix the empty component never displaying. It seems like it worked on native because it was calling _getRenderListEmptyComponent() in the calendarlist instead of passing in _getRenderListEmptyComponent.